tests/testthat/test-dfm.R

```diff
@@ -8,10 +8,10 @@ test_that("oldest dfm test", {
                               taxation="taxation",
                               taxregex="tax*",
                               country="united_states"))
-    dictDfm <- dfm(mycorpus, dictionary = mydict, valuetype = "glob")
-    dictDfm[1:10, ]
-    thesDfm <- dfm(mycorpus, thesaurus = mydict, valuetype = "glob")
-    thesDfm[1:10, (nfeature(thesDfm)-8) : nfeature(thesDfm)]
+    dictDfm <- dictDfm <- dfm(mycorpus, dictionary = mydict, valuetype = "glob")
+    dictDfm <- dictDfm[1:10, ]
+    dictDfm <- thesDfm <- dfm(mycorpus, thesaurus = mydict, valuetype = "glob")
```

Comment:
These are the oldest test for dfm that were not isolated. I do not see whey they are there either. I assigned output values to variables only because I do not want to set dfm to be printed to my console.